spec/inflector_test_cases.cr

```diff
@@ -0,0 +1,295 @@
+module InflectorTestCases
```

Comment:
I think this could go in a `spec/support` folder since it's not an actual spec file. Does that sound good to you?